apps/project/graphql/types.py

```diff
@@ -32,6 +32,14 @@ class ProjectTileServerCommonConfig: ...
 class ProjectTileServerConfig: ...
 
 
+@strawberry.experimental.pydantic.type(model=completeness_project.VectorTileServerConfig, all_fields=True)
+class ProjectVectorTileServerConfig: ...
```

Comment:
To avoid name clash in the future, let's add `Completeness` prefix

Reply:
This is a generic input so maybe we should move this to utils later.